{{cookiecutter.project_name}}/.github/workflows/test.yaml

```diff
@@ -27,11 +27,13 @@ jobs:
         include:
           - os: ubuntu-latest
             python: "3.10"
+            deps: stable
           - os: ubuntu-latest
-            python: "3.12"
+            python: "3.13"
+            deps: stable
           - os: ubuntu-latest
-            python: "3.12"
-            pip-flags: "--pre"
+            python: "3.13"
+            deps: pre
             name: PRE-RELEASE DEPENDENCIES
```

Comment:
To keep that in sync with `pyproject.toml`, maybe we could get this from `hatch env show --json`?